examples/CMakeLists.txt

```diff
@@ -1,128 +1,165 @@
-cmake_minimum_required(VERSION 3.2)
+cmake_minimum_required(VERSION 3.14)
 project(OpenDIS-Examples)
 
-## Libraries
-
-find_package(OpenDIS6)
-
-## Example Programs
-include_directories(.)
-
 # if Windows add M_PI definition
 # - issues occurred during testing in Visual Studio
 if (WIN32)
   add_definitions(/DM_PI=3.14159265358979323846)
 endif (WIN32)
 
-# create list of ExampleSender source files
-file(GLOB EX_SENDER_SOURCES
-  "main.cpp"
-  "Connection.cpp"
-  "Utils.cpp"
-  "Timer.cpp"
+find_package(SDL2 REQUIRED)
+find_package(SDL2_net REQUIRED)
+
+# Define common library
+add_library(example_common SHARED)
+# Link OpenDIS into ExampleSender
+target_link_libraries(example_common PUBLIC ${SDL2_LIBRARIES} ${SDL_NET_LIBRARIES})
```

Comment:
I tried compiling this and got errors when mentioning undefined reference to SDLNet_<XXX>.
You need to change `SDL_NET_LIBRARIES` to `SDL2_NET_LIBRARIES`.
Same with line 21 below, `SDL_NET_INCLUDE_DIRS` to  `SDL2_NET_INCLUDE_DIRS`.

Which is odd because I thought remembered SDL2_net being weird, in that it didn't need the explicitly state the 2, but I gave it a go and it compiled.

Reply:
Good catch, this must have been an issue with my local cache thinking it was working when it should have failed.